tools/tests/styles/rangeslider.css

```diff
@@ -0,0 +1,161 @@
+.horizontal-rangeslider-wrapper {
+    width: 40vh;
+    display: flex;
+    height: 20px;
+    display: flex;
+    align-items: center;
+    margin: 10vh;
+}
+
+.horizontal-rangeslider{
+    position: relative;
+    width: 100%;
+    height: 5px;
+    background-color:rgba(0, 0, 0, 0.3);
+    display: flex;
+    align-items: center;
+}
+
+.horizontal-rangeslider-handle {
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    background-color: #00ccff;
+    cursor: pointer;
+    z-index: 2;
+}
+
+.horizontal-rangeslider-bar {
+    position: absolute;
+    top: 0px;
+    left: 0px;
+    height: 100%;
+    width: 50%;
+    background-color: #0085a7;
+}
+
+.horizontal-rangeslider-grid {
+    display: flex;
+    flex-direction: row;
+    position: absolute;
+    top: 100%;
+    width: 100%;
+    justify-content: space-between;
+}
+
+.horizontal-grid-pol-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 2px;
+}
+
+.horizontal-grid-pol {
+    height: 10px;
+    width: 2px;
+    background-color: black;
+}
+
+.horizontal-grid-text {
+    color: black;
+    font-size: 0.8rem;
+}
+
+.horizontal-pol-without-text {
+    height: 5px;
+}
+
+.horizontal-rangeslider-thumb {
+    position: absolute;
+    top: -800%;

```

Comment:
Update here also and in the other places where we commented in the rangeslider.css of the component itself.